tools/experiments/experiments.js

```diff
@@ -505,6 +501,7 @@ function getAmpConfig() {
     // Cache bust, so we immediately reflect AMP_CANARY cookie changes.
     xhr.open('GET', '/v0.js?' + Math.random(), true);
     xhr.send(null);
+    resolve();
```

Comment:
this doesn't seem right. It would resolve the promise immediately but we want it resolved in `load`. Use `Deferred`?

Reply:
Ooops. I misread this section. Converted this to use `Deferred`.